Dockerfile

```diff
@@ -35,8 +32,6 @@ VOLUME [ "/data", "/input" ]
 COPY docker/entrypoint.sh /entrypoint.sh
 ENTRYPOINT [ "/entrypoint.sh" ]
 
-# Expose and Run using supervisor
-COPY docker/supervisord.conf /supervisord.conf
+# Expose and Run in gunicorn
```

Comment:
why have you decided to drop "supervisord-gunicorn" combination and run server just in gunicorn?